ansible/playbooks/aix.yml

```diff
@@ -34,7 +34,7 @@
     ################################
     - name: Download AIX filesystem configuration script
       get_url:
-        url: https://raw.githubusercontent.com/AdoptOpenJDK/openjdk-infrastructure/master/ansible/playbooks/scripts/AIX_filesystem_config.sh
+        url: https://raw.githubusercontent.com/jdekonin/openjdk-infrastructure/master/ansible/playbooks/scripts/AIX_filesystem_config.sh
```

Comment:
Why are we pointing to your username?

Reply:
Its the only way I could test, this would need to be changed prior to commit.

I have tested against my own fork and this works only on the 7.1.4 distro as there are version specific rpms.  Thats a future todo, if/when further versions are required.

I also encountered an issue with installing/upgrading the zlib rpm.  The version (1.2.11-1.aix5.1) commented out is required for openjdk to build, but that version breaks python.  My current answer for that is it needs to be performed manually.  Not ideal, but then at least the rest of the system setup can be performed via ansible.